procrastinate/sql/schema.sql

```diff
@@ -198,12 +203,12 @@ AS $$
 DECLARE
```

Comment:
procrastinate_fetch_job above hasn't been updated. This means that the second you request a job abortion, the next job with the same lock will start.

Reply:
Unfortunately, I don't fully understand your comment 🤷‍♂️. Do I have to change something in `procrastinate_fetch_job`, too?

Reply:
I think so, here's what an integration test for the correct behaviour would look like (I'm on my phone, so this is pseudocode)

```python
@app.task(look="foo")
def task():
    pass

job1 = task.defer()
job2 = task.defer()

fetch_job1 = app.job_manager.fetch_job()
fetch_job1.cancel(abort=True)

assert app.job_manager.fetch_job() is None
```

When a job is aborting, it's not finished yet, locks should still apply.

Reply:
It's amazing what you can write on your mobile 😃. Of course, I totally forgot the locks. Thanks a lot, that would have been a nasty bug. I hope I fixed it now.